Pod/Classes/VOKCoreDataManager.m

```diff
@@ -319,46 +296,20 @@ - (NSArray *)importArray:(NSArray *)inputArray forClass:(Class)objectClass withC
         } else {
             returnObject = [self managedObjectOfClass:objectClass inContext:contextOrNil];
             [mapper setInformationFromDictionary:inputDict forManagedObject:returnObject];
+            if (inputKey) {
+                [existingObjectArray addObject:returnObject];
+                [existingUniqueKeys addObject:inputKey];
```

Comment:
You get to this branch if, on line 284, the first part `inputKey` is true (non-`nil`) and the second part `[existingUniqueKeys containsObject:inputKey]` is false; that means that `matchingObjects` remains uninitialized form line 282 through to 288, so `matchingObjectsCount` is `[nil count]`, which is `0`, so the `if` statement on 290 is not true, leading to this `else` case, starting on 296.